core/src/main/java/com/scalar/db/util/ScalarDbUtils.java

```diff
@@ -249,4 +250,23 @@ public static boolean isRelational(Scan scan) {
     return scan instanceof ScanAll
         && (!scan.getOrderings().isEmpty() || !scan.getConjunctions().isEmpty());
   }
+
+  public static JDBCType getJdbcType(int sqlTypes) {
```

Comment:
> int sqlTypes

`sqlType` is better?

Reply:
Ahh, sqlTypes comes from `java.sql.Types`. Still weird? How about adding usage comments?

Reply:
Yeah, just looked at `java.sql.Types`. It consists of many `int` constant values (e.g., `public final static int SMALLINT =   5`.) So it's a set of integers (SQL types) in a sense.

`com.scalar.db.util.ScalarDbUtils#getJdbcType` seems to find the corresponding type out of the set of SQL types (`java.sql.Types`) by a single key `int sqlType(s)`. So I think we don't need to treat `int sqlType(s)` as plural.

Reply:
Fixed in 0d3a2f8.